statusroutes_test.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/gorilla/mux"
+	"github.com/stretchr/testify/require"
+)
+
+func TestStatusRoutes(testCase *testing.T) {
+	testRouter := mux.NewRouter()
+	serviceName := "%CUSTOM_PLUGIN_SERVICE_NAME%"
+	serviceVersion := "0.0.0"
+	StatusRoutes(testRouter, serviceName, serviceVersion)
+
+	testCase.Run("/-/healthz - ok", func(assert *testing.T) {
+		expectedResponse := fmt.Sprintf("{\"status\":\"OK\",\"name\":\"%s\",\"version\":\"%s\"}", serviceName, serviceVersion)
+		responseRecorder := httptest.NewRecorder()
+		request, error := http.NewRequest(http.MethodGet, "/-/healthz", nil)
+		if error != nil {
+			assert.Fatal("Error creating the /-/healthz request")
+		}
+		testRouter.ServeHTTP(responseRecorder, request)
+		statusCode := responseRecorder.Result().StatusCode
+		require.Equal(assert, http.StatusOK, statusCode, "The response statusCode should be 200")
+
+		rawBody := responseRecorder.Result().Body
+		buffer := new(bytes.Buffer)
+		buffer.ReadFrom(rawBody)
```

Comment:
This could be replaced with
```suggestion
		rawBody, err := ioutil.ReadAll(r)
		require.NoError(t, err)
```
In this way rawBody is an array of bytes. To convert to string, you could use: `string(rawBody)`

Reply:
Yes I was converting it to string into the assertion by using the `buffer.String()` method.
Anyway, I changed it to with your suggestion, it's more clean.